gradle/shared-with-buildSrc/code-quality-configuration.gradle.kts

```diff
@@ -37,6 +37,7 @@ fun Project.configureCheckstyle(codeQualityConfigDir: File) {
     val checkStyleConfigDir = codeQualityConfigDir.resolve("checkstyle")
     configure<CheckstyleExtension> {
         configDir = checkStyleConfigDir
+        toolVersion = "6.19"
```

Comment:
@oehme I had to set the `toolVersion` for our own build explicitly because our Checkstyle config is not compatible with 8.12.

Reply:
What's stopping us from changing the config and tool version at the same time?

Reply:
I tried that but it made some checkstyle tasks fail and I didn't want to get sidetracked. I'll take a closer look once this is merged.